_rules/80f0bf.md

```diff
@@ -0,0 +1,112 @@
+---
+name: No auto-play audio
+test_type: composite
+
+description: |  
+  This rule checks that auto-play audio does not last for more than 3 seconds, or it has a control mechanism.
+
+success_criterion: 
+- 1.4.2 # Audio Control
+
+atomic_rules:
+- 95e043
+- aaa1bf
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>` `<video>` or `<source>` elements, with a `src` attribute referencing content with a duration of more than 3 seconds or with a `loop` attribute equal to `true`, that has an `autoplay` attribute equal to `true`, and that has either a `paused` or `muted` attribute equal to `false`.
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+- Auto-play audio does not last more than 3 seconds
+- Auto-play audio has a control mechanism
+ 
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
```

Comment:
`<video>` and `<audio>` elements are not native accessible elements. In some UA/AT combinaisons, controls are not reachable by using a keyboard only (play/pause can't be accessed nor triggered) and even more, with some AT the video or audio element is not announced (it just says "empty" when reaching a video).

For example : in some UA, controls doesn't exist in the accessibility tree. In contrast, in others every controls of the elements exists in the accessibility and has an accessible name (and is keyboard operable). 

The mozilla documentation gives some informations : https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia (even if I think some informations needs updates)

I think we need so specify that lack of support.

Reply:
Added this.